packages/angular/ssr/test/routes/router_spec.ts

```diff
@@ -6,8 +6,13 @@
  * found in the LICENSE file at https://angular.dev/license
  */
 
+/* eslint-disable import/no-unassigned-import */
+import '@angular/compiler';
```

Comment:
Nit: Can you add a comment about the side effect we're relying on here? I assume this is needed for the JIT compiler, but I'm surprised there's a load-bearing top-level side effect we can depend on in this context.